### DISCUSSION TITLE
Temp Email

### DISCUSSION THREAD

**Discussion by jhuynh08** [inbucket/inbucket#303] [Q&A] [answered]
Hello,

I tried testing using inbucket as an temp email service but I can't seem to receive any email. I took the route of not fronting it with a mail server like postfix so I understand that there's no authentication. I set the accepted domains to my desired domain and also the MX records for my DNS and tried to email from my gmail account to my test domain with no luck.

Is there anything else that I could be missing?

My env are shown below and I'm currently deploying on kubernetes:
```
      INBUCKET_STORAGE_PARAMS: maxkb:50240
      INBUCKET_WEB_UIDIR: ui
      INBUCKET_WEB_GREETINGFILE: "defaults/greeting.html"
      INBUCKET_MAILBOXNAMING: full
      INBUCKET_SMTP_DOMAIN: "tempmail.gexhub.co"
      INBUCKET_SMTP_ACCEPTDOMAINS: "mail.gexhub.co"
```

Thank you!

**Comment by jhillyerd** [comment, marked as answer]
Thanks for trying out Inbucket!

A couple things to check:

1. I see tempmail.gexhub.co resolves to two different IPs -- is that intentional?  Inbucket won't sync between multiple instances (and doesn't support shared storage), so you'd need to check both web UIs independently.
2. You should be able to see a dump of the connections & traffic in inbuckets output if you turn the log level up to debug.  This would help you see if the connection is being made but something is preventing the mail from going through.

**Reply by jhuynh08** [reply]
Ah let me check this again. I may have missed something in my configs.
